test/runtests.jl

```diff
@@ -2,9 +2,16 @@ using Fleck
 using SafeTestsets
 using Test
 
-@testset "Fleck.jl" begin
+@testset "test_prefixsearch.jl" begin
```

Comment:
Ah, you want nested testsets. Sure. Sure.

Reply:
I defer to your preferred convention. This is simply what I'm used to from collaborating with the AlgJulia folks.

Reply:
Once you're in the world of `@testset`, it doesn't matter whether it's nested. There is an advantage for localization of variables, but there is also a disadvantage for creating fixtures. Julia's testing environment is weak compared to Python's. I say we go with this suggestion. Always feels better to be modern.

Reply:
They may be following the SciML style guide: https://github.com/SciML/SciMLStyle#tests-and-continuous-integration